crates/sui-indexer-alt-framework/src/pipeline/concurrent/mod.rs

```diff
@@ -257,5 +257,9 @@ pub(crate) fn pipeline<H: Handler + Send + Sync + 'static>(
 }
 
 const fn max_chunk_rows<H: Handler>() -> usize {
+    // Handle division by zero
+    if H::Value::FIELD_COUNT == 0 {
+        return 0;
```

Comment:
This probably won't happen in real life since I don't think any value will have zero fields, but it can happen in tests and just in case.